Lib/asyncio/selector_events.py

```diff
@@ -266,6 +266,7 @@ def _add_reader(self, fd, callback, *args):
                                   (handle, writer))
             if reader is not None:
                 reader.cancel()
+        return handle
```

Comment:
Strictly speaking this is an API change -- it looks like `loop.add_reader` et al currently returns `None`, and with this change they will return `events.Handle`, right?

Reply:
Yes